src/Corax/Indexing/IndexWriter.cs

```diff
@@ -944,9 +945,9 @@ private bool TryGetEntryTermId(Slice fieldName, ReadOnlySpan<byte> term, out lon
             return fieldTree.TryGetValue(termValue, out idInTree);
         }
 
-        public void Commit() => Commit<EmptyStatsScope>(default);
+        public void Commit() => Commit<EmptyStatsScope>(stats: default, token: CancellationToken.None);
```

Comment:
why aren't we forcing a cancellation token here? Who is calling this?

Reply:
I've added to ability to pass token; however this overload is used only by tests and `FlushBatch`